### PR TITLE
Add leather shields to the crafting calculator #4326

### PR DESCRIPTION
**Pull request by rbbi**
screenshot at jul 15 02-46-05

### CODE REVIEW

File: runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json [hunk 1]
```diff
@@ -408,6 +408,30 @@
...
+    {
+      "level": 62,
+      "icon": 22275,
+      "name": "Green D'hide Shield",
+      "xp": 124
+    },
+    {
+      "level": 69,
+      "icon": 22278,
+      "name": "Blue D'hide Shield",
```
Comment (deathbeam) [inline runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json]: The fields here are sorted by level so you should follow that convention

**Comment by Nightfirecat** [comment]
This is missing hard leather and snakeskin shields. Not sure why they were excluded.

**Comment by deathbeam** [comment]
Because noone ever crafted them most likely :d

**Comment by Nightfirecat** [comment]
inb4 "Hard leather and snakeskin shields missing from skill calc" issue :upside_down_face:

File: runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json [hunk 2]
```diff
@@ -263,6 +263,12 @@
       "icon": 1660,
       "name": "Ruby Necklace",
       "xp": 75
+    },
+	{
```
Comment (Adam-) [inline runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json]: Format this consistently with the rest of the file.

Comment (rbbi) [reply runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json]: @Adam- same as #4334

**Comment by rbbi** [comment]
@deathbeam any update on this?

**Pull request merged**